reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+
+- Azure PowerShell (Az.*)
+- Active Directory
+- Many of the modules in Windows 10 and Windows Server (check with `Get-Module -ListAvailable`)
+
+For the current list of supported modules, see [PowerShell 7 module compatability](/powershell/scripting/whats-new/module-compatibility?view=powershell-7).
+
+[!NOTE]
+> On Windows, we've also added a **UseWindowsPowerShell** switch to Import-Module to ease the
+> transition to PowerShell 7 for those using incompatible modules. This switch creates a proxy
+> module in PowerShell 7 that uses a local Windows PowerShell process to implicitly run any cmdlets
+> contained in that module. For more information on this functionality, check out the Import-Module
+> documentation.
+
+For those modules still incompatible, we're working with a number of teams to add native PowerShell
+7 support, including Microsoft Graph, Office 365, and more.
+
+Azure Cloud Shell has already been updated to use PowerShell 7, and others like the .NET Core SDK
+Docker container images and Azure Functions will be updated soon.
+
+## Where is the ISE (Integrated Scripting Environment)?
+
+The Windows PowerShell Integrated Scripting Environment (ISE) is only supported in Windows
+PowerShell. To support the growing needs of the automation and scripting community, along with
+DevOps and developers, we recommend [Visual Studio Code (VSCode)](https://code.visualstudio.com/)
+and the [PowerShell Extension](https://code.visualstudio.com/docs/languages/powershell) for writing
+and debugging scripts and modules.
+
+To make the transition to Visual Studio Code easier, use the **Enable ISE Mode** function available
+in the the Command Palette. This function switches VSCode into an ISE-style layout. The ISE-style
+layout give you all the new features and capabilities of PowerShell in a familiar user experience.
+
+To switch between the default layout and the new ISE layout, press;
+
++ <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
+
+Type and select: **PowerShell:Enable ISE Mode**
+
+To set the layout to the original VSCode layout, open the Command Palette, type and select
+**PowerShell: Disable ISE Mode (restore to defaults)**
+
+For details about customizing the VSCode layout to ISE, see
+[How to Replicate the ISE Experience in Visual Studio Code](/powershell/scripting/components/vscode/how-to-replicate-the-ise-experience-in-vscode)
+
+> [!NOTE]
+> The PowerShell Team and its partners are actively working to improve the scripting
+> experience in the PowerShell extension for VSCode. We currently have no plans to update the ISE
+> with new features. In the latest versions of Windows 10 and Windows Server, the ISE is now a
+> user-uninstallable feature. We have no plans to make the ISE unavailable in the future.
+
+## Profiles
+
+A PowerShell profile is a script that executes when PowerShell starts. You can use the profile as a
+logon script to customize the environment. You can add commands, aliases, functions, variables,
```

Comment:
Only take this one if @sdwheeler says it's the right language. I just haven't seen/heard "logon script" in a very long while. 

```suggestion
login script to customize the environment. You can add commands, aliases, functions, variables,
```

Reply:
The correct term is "logon script". That matches the UI in Windows.